lib/utils.ts

```diff
@@ -1,4 +1,4 @@
-var packageInfo = require('../package.json');
+const packageInfo = require('../package.json');
```

Comment:
Can be converted to an import statement.
Will need `resolveJsonModule: true` in tsconfig.json